packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -67,16 +68,29 @@ export type TabsProps = {
    * Identifier for the selected tab.
    */
   value?: string;
+  /**
+   * Callback fired when the active tab is changed.
+   */
+  onChange?: (event: SyntheticEvent, value: string) => void;
 };
 
-const Tabs = ({ ariaLabel, initialValue, tabs, value }: TabsProps) => {
+const Tabs = ({
+  ariaLabel,
+  initialValue,
+  tabs,
+  value,
+  onChange,
+}: TabsProps) => {
   const [tabState, setTabState] = useState(initialValue ?? value ?? "0");
 
-  const onChange = useCallback(
-    (_event: React.SyntheticEvent, newState: string) => {
-      setTabState(newState);
+  const handleChange = useCallback(
+    (event: SyntheticEvent, value: string) => {
+      setTabState(value);
+      if (onChange) {
+        onChange(event, value);
+      }
```

Comment:
We can simplify this with `?.`:
```suggestion
      onChange?.(event, value);
```

I don't think it affects readability, but you can provide feedback.